src/execution_plan.h

```diff
@@ -260,6 +261,7 @@ typedef struct FlatExecutionPlan{
     FlatBasicStep onRegisteredStep;
     FlatBasicStep onUnpausedStep;
     long long executionMaxIdleTime;
+    ExecutionFlags flags;
```

Comment:
ExecutionFlags are for executions, lets create a FlatExecutionFlags

Reply:
ok
I'm leaving `EPTurnOnFlag` or rename too?